plugins/base/src/main/kotlin/renderers/html/htmlPreprocessors.kt

```diff
@@ -13,22 +13,24 @@ import org.jetbrains.dokka.plugability.DokkaContext
 import org.jetbrains.dokka.plugability.configuration
 import org.jetbrains.dokka.transformers.pages.PageTransformer
 
-object NavigationPageInstaller : PageTransformer {
+open class NavigationPageInstaller : PageTransformer {
```

Comment:
I have some doubts it this should have a separate interface

Reply:
Yes.